formats/json/commonMain/src/kotlinx/serialization/json/Json.kt

```diff
@@ -287,7 +287,7 @@ public class JsonBuilder internal constructor(json: Json) {
     public var prettyPrintIndent: String = json.configuration.prettyPrintIndent
 
     /**
-     * Enables coercing incorrect JSON values to the default property value in the following cases:
+     * Enables coercing incorrect JSON values to the default property value (if exists) in the following cases:
```

Comment:
M.b. `if specified`?

Reply:
I think `if specified` may be incorrectly read as `if coerceInputValues = true is specified`